packages/data/src/store/index.js

```diff
@@ -1,12 +1,44 @@
-/**
- * Internal dependencies
- */
-import reducer from './reducer';
-import * as selectors from './selectors';
-import * as actions from './actions';
-
-export default {
-	reducer,
-	actions,
-	selectors,
-};
+
+function createCoreDataStore( registry ) {
+	const getCoreDataSelector = ( selectorName ) => ( reducerKey, ...args ) => {
+		return registry.select( reducerKey )[ selectorName ]( ...args );
+	};
+
+	const getCoreDataAction = ( actionName ) => ( reducerKey, ...args ) => {
+		return registry.dispatch( reducerKey )[ actionName ]( ...args );
+	};
+
+	return {
+		getSelectors() {
+			return [
+				'getIsResolving',
+				'hasStartedResolution',
+				'hasFinishedResolution',
+				'isResolving',
+				'getCachedResolvers',
+			].reduce( ( memo, selectorName ) => ( {
+				...memo,
+				[ selectorName ]: getCoreDataSelector( selectorName ),
+			} ), {} );
+		},
+
+		getActions() {
+			return [
+				'startResolution',
+				'finishResolution',
+				'invalidateResolution',
+				'invalidateResolutionForStore',
+				'invalidateResolutionForStoreSelector',
+			].reduce( ( memo, actionName ) => ( {
+				...memo,
+				[ actionName ]: getCoreDataAction( actionName ),
+			} ), {} );
+		},
+
+		subscribe() {
+			return () => {};
+		},
```

Comment:
Because there's no store stored in this store that need to retrigger selectors, if a change in the resolution state happens, the corresponding store would have already triggered a "subscribe" call.

Reply:
> Because there's no store stored in this store that need to retrigger selectors, if a change in the resolution state happens, the corresponding store would have already triggered a "subscribe" call.

Can we add this as an inline code comment for the benefit of the next maintainer?

Reply:
Would a more accurate rendition include "Because there's no store stored in this _registry_..."?